src/types/transaction.ts

```diff
@@ -41,6 +41,8 @@ export interface SendTransactionParams {
   maxFee?: AlgoAmount
   /** The maximum number of rounds to wait for confirmation, only applies if `skipWaiting` is `undefined` or `false`, default: wait up to 5 rounds */
   maxRoundsToWaitForConfirmation?: number
+  /** Whether to use simulate to automatically pack app call resources into the txn objects. Defaults to true when there are app calls in the group.  */
+  packResources?: boolean
```

Comment:
I think it should be called `packAppCallResources`

I also wonder if `undefined` should default to `true`, it's a bit of a breaking change in behaviour, but I guess it's low risk assuming that if you correctly specify the resources I assume they aren't returned as `unnamed resources`?

Reply:
Yeah my thinking is that it only changes the behavior of app calls that would've failed anyways.